examples/README.md

```diff
@@ -0,0 +1,31 @@
+# isomdl exmples
+
+# Connection and data exchange between a device and a reader
```

Comment:
Would prefer the use of "simulated device and reader" throughout this example and its docs because there is not actually a device nor reader, and there's no QR/BLE exchange. Only DeviceEngagement and in-memory channels.

Reply:
sure, changed